### PR TITLE
Revert "Adjust allowed mutations, summary script, bgzip inputs, pysam compatibility"

### PR DESCRIPTION
**Pull request by jwerner-dkfz**
Reverts eilslabs/DKFZBiasFilter#4

found one problem with the pull request. 

@schmattes42 please review

### CODE REVIEW

File: scripts/readbiasfunctions.py [hunk 1]
```diff
@@ -1,838 +1,834 @@
...
 """
-import gzip
 import sys
 import pysam
+import numpy as np
+import math
 from scipy.stats import binom
 
-# Also adjust in readbiasplots to synchronize
-POSSIBLE_MUTATIONS = ["CA", "CG", "CT", "TA", "TC", "TG", "GT", "GA"]
```
Comment (schmattes42) [inline scripts/readbiasfunctions.py:11]: Redefine!
POSSIBLE_MUTATIONS = ["CA", "CG", "CT", "TA", "TC", "TG"]

File: scripts/readbiasplots.py [hunk 2]
```diff
@@ -9,269 +9,265 @@
 import matplotlib.pyplot as plt
 import matplotlib.gridspec as gridspec
 
-# Also adjust in readbiasfunctions to synchronize
-POSSIBLE_MUTATIONS = ["CA", "CG", "CT", "TA", "TC", "TG", "GT", "GA"]
```
Comment (schmattes42) [inline scripts/readbiasplots.py:13]: Redefine!
POSSIBLE_MUTATIONS = ["CA", "CG", "CT", "TA", "TC", "TG"]

**Review by schmattes42 — COMMENTED** [review]
Looks almost all good. The problem with the pull request of chpmanb was the introduction of redundancy in POSSIBLE_MUTATIONS = ["CA", "CG", "CT", "TA", "TC", "TG", "GT", "GA"]. The newly introduced possible mutations are GT, and GA, that are the same as CA, and CT, since we merge complementary mutations.

**Comment by chapmanb** [comment]
Thank you for the review and happy to make this change. I'm running into an issue though because of this reversion. I'm not able to figure out how to rebase against the current head without losing my changes (since they got incoporated then removed). The cleanest way for you to fix might be to grab the original diff:

https://patch-diff.githubusercontent.com/raw/eilslabs/DKFZBiasFilter/pull/4.diff

Apply that to master, and then apply a commit that fixes the duplicate complementary mutations. Would this work for you? Alternatively I could remove my old branch, start a new one and apply this flat patch for a PR if you prefer that.

Thanks again for getting this in and apologies about the extra work.

**Pull request merged**